src/patternfly/components/Select/select.scss

```diff
@@ -438,13 +432,6 @@
   background-color: inherit;
 }
 
-.pf-c-select__separator {
```

Comment:
Hmm... we're taking this away and not replacing it. I think it would be better to style `pf-c-divider` here like you have in the other dropdown components, and use a container around the divider between the search box and rest of the menu items to unset the margin on the divider there.

I'm also not sure that needs to be a divider as it's just a visual treatment and doesn't convey anything. The meaning is the same whether it's there or not. IMO it could (should?) just be a border.